website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -20,6 +20,12 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## September 2024
 
+- **New**: You can set descriptions in your YAML files to document [singular data tests](/docs/build/data-tests#singular-tests).
+- **New**: The snapshot configuration can now be set in the schema YAML file, making it easier to manage and organize your project’s configuration in one place. 
+- **New**: You will now get a warning if the data type of the `updated_at` column does not match the adapter-configured default.
+- **New**: You can now define the names of the columns created in the [snapshot table](/docs/build/snapshots#snapshot-meta-fields) using custom configurations.
+- **New**: dbt `test` now enables passing values like `unit_tests`, `tests`, `--resource-type` and `--exclude-resource-type` for CLI flags and `DBT_RESOURCE_TYPES` and `DBT_EXCLUDE_RESOURCE_TYPES)`  for environment variables.
+- **Enhancement**: When [defining constraints](/reference/resource-properties/constraints#defining-constraints) you can use the improved syntax to define foreign keys, which uses `ref`. The recommended syntax means dbt automatically resolves references to other models, ensuring they capture dependencies and work across different environments.
```

Comment:
```suggestion
- **Enhancement**: When [defining constraints](/reference/resource-properties/constraints#defining-constraints) you can now define foreign keys using `ref`, so that dbt will automatically resolves references to other models, ensuring they capture dependencies and work across different environments.
```

@jtcohen6 lmk if this looks good to you